src/styles/main.scss

```
@@ -0,0 +1,92 @@
+@import "./mixin.scss";
+
+.container {
+  position: relative;
+  width: 900px;
+  height: 700px;
+  margin: 0 auto;
+}
+
+.clock {
+  margin: 0 auto;
+  position: relative;
+  width: 620px;
+  height: 620px;
+  border-radius: 50%;
+  border: 10px solid black;
```

don't forget to create variables for all colors
```suggestion
  border: 10px solid $c-black;
```